.github/workflows/stale.yaml

```diff
@@ -9,10 +9,11 @@ jobs:
     runs-on: ubuntu-18.04
     if: github.repository == 'rook/rook'
     steps:
-    - uses: actions/stale@v3
+    - uses: actions/stale@v4
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        days-before-stale: 90
+        days-before-issue-stale: 45
```

Comment:
If it's too short, I don't think it gives a good impression to the community that we're open to investigating issues. IMO it's better to be generous with time.

Reply:
Is a month and a half not generous enough? Also, we can still put a keepalive flag if we need to.

Reply:
I vote for 60 days for issues, 30 days for PRs.